stubs/mock/mock/backports.pyi

```diff
@@ -0,0 +1,11 @@
+import sys
+
+if sys.version_info[:2] >= (3, 8):
+    from asyncio import iscoroutinefunction as iscoroutinefunction  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
What is pyright unhappy about here?

Reply:
Arg names / signatures do not match: https://github.com/python/typeshed/actions/runs/3272486451/jobs/5383555539

Reply:
It shouldn't matter because they're in different version branches. The problem is that you have a `[:2]` on the version check, that shouldn't be there. I'm surprised flake8-pyi doesn't complain.